src/less/multi-avatars.less

```diff
@@ -0,0 +1,32 @@
+.multi-avatars {
+  display: inline-flex;
+  flex-direction: row-reverse;
+  width: 40px;
```

Comment:
Can we confirm with design/product that these conform to the base size of avatars? In the protoype it seems that the height of the multi-avatar is the same as a regular avatar with the default size. (which i think is 60px).

Reply:
its 40px 
![image](https://user-images.githubusercontent.com/16889026/65548779-28384280-deea-11e9-9bb2-84be2fd09f57.png)